src/components/index.js

```diff
@@ -3,3 +3,4 @@ export { Sidebar } from "./sidebar/sidebar";
 export { Card } from "./card/card";
 export { Chips } from "./filterChips/chips";
 export { chipsData } from "./filterChips/chipsData";
```

Comment:
seems like you haven't used 'chip' here, so can be removed

Reply:
its used in explore page